applications/ShapeOptimizationApplication/custom_utilities/geometry_utilities.h

```diff
@@ -196,6 +206,68 @@ class GeometryUtilities
     	KRATOS_CATCH("");
     }
 
+    // --------------------------------------------------------------------------
+    void ComputeProjectedDistancesToBoundingModelPart(
+        ModelPart& rBoundingModelPart,
```

Comment:
The function assumes that `rBoundingModelPart` is a surface part. So maybe it makes sense to include a quick check in the beginning, to make sure it does not contain volume elements.

Reply:
done